addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -906,11 +1034,55 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         field.records = valueList;
         await this._replaceField(field);
     },
-
+    /**
+     * @param {string} value visible, hidden or visible only if
+     */
+    visibility: function (previewMode, value, params) {
+        this._visibility(value);
+    },
+    /**
+     * @param {string} value the value that changes the visibility
+     */
+    visibilityCondition: function (previewMode, value = '', params) {
+        this._visibilityCondition(value);
+    },
+    /**
+     * @param {string} value the way that the input value and the visibility-condition will be compared
+     */
+    visibilityComparaison: function (previewMode, value, params) {
+        this._visibilityComparaison(value);
+    },
+    /**
+     * @param {string} value when comparator is between or !between, value is the max date
+     */
+    visibilityBetween: function (previewMode, value, params) {
+        this._visibilityBetween(value);
+    },
+    /**
+     * @param {string} value the id of the input that influences the visibility of the current field
+     */
+    visibilityDependency: function (previewMode, value, params) {
+        this._visibilityDependency(value);
+    },
```

Comment:
I have not read this whole option code, first you have to refactor with what exists:
- You don't need a equivalent private method for most of those methods since you call them only here
- Some are strictly equal to `selectDataAttribute` which you can use

Try to review and simplify this according to that and I'll review :wink: